changelog/2.006.dd

```diff
@@ -31,8 +31,7 @@ $(WHATSNEW
 	$(LI Added the
 	$(LINK2 $(ROOT_DIR)phobos/std_variant.html, std.variant) module.)
 	$(LI Incorporated many of the Tango GC structural differences (much more to go still).)
-	$(LI Added the
-	$(LINK2 $(ROOT_DIR)phobos/std_contracts.html, std.contracts) module.)
+	$(LI Added the $(D std.contracts) module.)
```

Comment:
> If we can upstream Seb's doc archive, then this certainly sounds like a very good idea.

Do you mean moving the repo to `dlang`? Or moving it under the `dlang.org` domain? I wouldn't mind about both. After all, it's just a raw dump (deployed on netlify)